src/main/resources/ca/on/oicr/gsi/somnus/ui.js

```diff
@@ -88,6 +101,14 @@ export default function(form, environments, services) {
       alert("Invalid input");
       return;
     }
+
+    if(units.value === "hours" || units.value === "days" || units.value === "seconds"){
+      var sanityCheck = prompt("Please confirm the units you selected.");
```

Comment:
What do you think about `"Please confirm the units you selected (seconds, minutes, hours, days)."`

Reply:
Sure.